flink-connectors/flink-connector-hive/src/test/resources/endpoint/select.q

```diff
@@ -47,7 +47,7 @@ CREATE TABLE hive_types_table (
     f7 DECIMAL(20, 8),
     f8 STRING,
     f9 VARCHAR(20),
-    f10 TIMESTAMP(9),
+    f10 TIMESTAMP,
```

Comment:
Hive dialect doesn't support `TIMESTAMP(9)`. The origin code will fall to Flink default parser which  is not what we expect.